SQL/Archive/2017-11-01-use_consent_to_database.sql

```diff
@@ -29,13 +29,13 @@ CREATE TABLE `candidate_consent_type_rel` (
 
 CREATE TABLE `candidate_consent_type_history` (
   `CandidateConsentHistoryID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `EntryDate` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
```

Comment:
@PapillonMcGill did i get it right this time? lol

Reply:
Yup!